pages/storylets/harlowe/harlowe_storylets.md

```diff
@@ -0,0 +1,78 @@
+# "Storylets": Harlowe (v3.2)
+
+## Summary
+
+Storylets are [a different way of approaching nonlinear storytelling](https://emshort.blog/2019/11/29/storylets-you-want-them/). Instead of linking from one passage to another in Twine, storylets are defined by what are called its *requirements* and *content*.
+
+The metaphor of a deck of cards is often useful in understanding storylets. A card is able to be drawn from a deck if its requirements are met. Through defining many cards (passages in Twine) and conditions under which they can be used (their requirements), dynamic storytelling can take place as different arrangements become possible through writing simple rules for each card.
+
+The [`(storylet:)` macro](https://twine2.neocities.org/#macro_storylet) defines a passage as a storylet in Harlowe. It should appear as either the first line or as near to the top of the passage as possible. To define if it is available (called "open" in Harlowe) or not, it uses a [`when` lambda](https://twine2.neocities.org/#type_lambda) for determining, in the future, if the requirements are met using story and temporary variables.
+
```

Comment:
> To define if it is available (called "open" in Harlowe) or not, it uses a `when` lambda for determining, in the future, if the requirements are met using story and temporary variables.

While this statement is factually true, and it includes two important terms ('open' and 'when') related to storylets, I wonder if it is too technical in wording for those with less programming experience and if not enough emphasise is given to the key 'open' term?

The following is not a replacement suggestion, just an example of one way to add emphasise to the 'open' term.
> A Harlowe storylet is considered 'open' when its lamba conditional expression evaluates to true.

Reply:
If you're concerned about the 'open' language, maybe try something like 
> A Harlowe storylet is considered **open**, meaning it is available to be "drawn," when its lamba conditional expression evaluates to true.